### DISCUSSION TITLE
Measure unique users initiating custom events

### DISCUSSION THREAD

**Discussion by legkikh** [plausible/analytics#3239] [Anything Else]
Hello,

I'm interested in measuring the engagement of users in my app. Currently, I track the total number of significant user actions, but I'd like to consider **only one event per user per day**. Is this functionality already available, or is there a way to achieve this using Plausible?

**Comment by metmarkosaric** [comment]
hi @legkikh! when you set up a custom event we show both unique and total conversions. so for instance if you have 1 visitor converting on the same event 5 times in a day, on the dashboard you'll see one unique conversion and 5 total conversion for that one visitor on that event
